homeassistant/components/alexa/smart_home.py

```diff
@@ -169,6 +170,17 @@ def async_api_entity_wrapper(hass, request):
     return async_api_entity_wrapper
 
 
+@HANDLERS.register(('Alexa.SceneController', 'Activate'))
+@extract_entity
+@asyncio.coroutine
+def async_api_activate(hass, request, entity):
+    """Process an activate request."""
+    service = entity.entity_id.split('.')[1]
+    yield from hass.services.async_call(entity.domain, service, blocking=True)
+
+    return api_message(request)
```

Comment:
This is incorrect. You need to return an `ActivationStarted` event as per the docs: https://developer.amazon.com/docs/device-apis/alexa-scenecontroller.html#activationstarted

I also think that scripts shouldn't be scenes, because a script that has a delay can actually be cancelled.